src/Altinn.App.Api/Altinn.App.Api.csproj

```diff
@@ -3,8 +3,8 @@
   <PropertyGroup>
     <TargetFramework>net5.0</TargetFramework>
     <OutputType>Library</OutputType>
-    <Version>5.0.0</Version>
-    <AssemblyVersion>5.0.0.0</AssemblyVersion>
+    <Version>5.1.0</Version>
```

Comment:
This should be 5.1.0-alpha.1 until the package is tested and verified with test apps.